src/polkadot/polkadotEngine.ts

```diff
@@ -243,7 +243,7 @@ export class PolkadotEngine extends CurrencyEngine<PolkadotTools> {
     this.addToLoop('queryTransactions', TRANSACTION_POLL_MILLISECONDS).catch(
       () => {}
     )
-    super.startEngine().catch(() => {})
+    await super.startEngine().catch(() => {})
```

Comment:
Please remove this catch. We want to fail if the super fails, since we are pretty much dead in the water.